meson.build

```diff
@@ -153,6 +153,49 @@ install_data(
     install_dir: join_paths(get_option('out'), 'etc/xdg/waybar')
 )
 
+scdoc = dependency('scdoc', version: '>=1.9.2', native: true, required: false)
+
+if scdoc.found()
+    scdoc_prog = find_program(scdoc.get_pkgconfig_variable('scdoc'), native: true)
+    sh = find_program('sh', native: true)
+    mandir = get_option('mandir')
+    man_files = [
+        'waybar.5.scd',
+        'waybar-backlight.5.scd',
+        'waybar-battery.5.scd',
+        'waybar-clock.5.scd',
+        'waybar-cpu.5.scd',
+        'waybar-custom.5.scd',
+        'waybar-idle-inhibitor.5.scd',
+        'waybar-memory.5.scd',
+        'waybar-mpd.5.scd',
+        'waybar-network.5.scd',
+        'waybar-pulseaudio.5.scd',
+        'waybar-sway-mode.5.scd',
+        'waybar-sway-window.5.scd',
+        'waybar-sway-workspaces.5.scd',
+        'waybar-temperature.5.scd',
+        'waybar-tray.5.scd',
+    ]
+
+    foreach filename : man_files
+        topic = filename.split('.')[-3].split('/')[-1]
+        section = filename.split('.')[-2]
+        output = '@0@.@1@'.format(topic, section)
```

Comment:
What the output for the `waybar.5.scd' file?

Reply:
As in the path? If you generate the build with `meson build && ninja -C build` then the man pages will be in `build`.

Installing with `ninja -C build install` gives the following output though:

```
sudo ninja -C build install
[sudo] password for root:
ninja: Entering directory `build'
[0/1] Installing files.
Installing waybar to /usr/local/bin
Installing waybar.5 to /usr/local/share/man/man5
Installing waybar-backlight.5 to /usr/local/share/man/man5
Installing waybar-battery.5 to /usr/local/share/man/man5
Installing waybar-clock.5 to /usr/local/share/man/man5
Installing waybar-cpu.5 to /usr/local/share/man/man5
Installing waybar-custom.5 to /usr/local/share/man/man5
Installing waybar-idle-inhibitor.5 to /usr/local/share/man/man5
Installing waybar-memory.5 to /usr/local/share/man/man5
Installing waybar-mpd.5 to /usr/local/share/man/man5
Installing waybar-network.5 to /usr/local/share/man/man5
Installing waybar-pulseaudio.5 to /usr/local/share/man/man5
Installing waybar-sway-mode.5 to /usr/local/share/man/man5
Installing waybar-sway-window.5 to /usr/local/share/man/man5
Installing waybar-sway-workspaces.5 to /usr/local/share/man/man5
Installing waybar-temperature.5 to /usr/local/share/man/man5
Installing waybar-tray.5 to /usr/local/share/man/man5
Installing /home/michael/Development/waybar/./resources/config to /etc/xdg/waybar
Installing /home/michael/Development/waybar/./resources/style.css to /etc/xdg/waybar
```